reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -78,8 +79,8 @@ def iterative_refinement(self, wiener_small_number=0.01, count_norm_const=1):
         ctfs = self.build_ctf_array()
         ctfs_1, ctfs_2 = IterativeRefinement.split_array(ctfs)
 
-        particles_f_1 = IterativeRefinement.fft_3d(particles_1)
-        particles_f_2 = IterativeRefinement.fft_3d(particles_2)
+        particles_f_1 = primal_to_fourier_3D(particles_1)
```

Comment:
nope... you want 2d fft of the particles

and 3d fft of the 3d map

Reply:
oops